examples/chatbot.py

```diff
@@ -0,0 +1,191 @@
+import os
+import openai
+import nest_asyncio
+import certifi
+import nltk
+from pathlib import Path
+from llama_index.readers.file import UnstructuredReader
+from llama_index.core import VectorStoreIndex, StorageContext, Settings, load_index_from_storage
+from llama_index.core.tools import QueryEngineTool, ToolMetadata
+from llama_index.llms.openai import OpenAI
+from llama_index.core.query_engine import SubQuestionQueryEngine
+from llama_index.agent.openai import OpenAIAgent
+from aimon.client import Client
+from llama_index.embeddings.openai import OpenAIEmbedding
+from dotenv import load_dotenv
+import requests
+import json
+from aimon import Config
+import logging
+import time
+
+load_dotenv()
```

Comment:
Lets remove the need to get this from .env and get the required credentials from the UI.